apps/molgenis-components/src/client/client.ts

```diff
@@ -375,7 +375,12 @@ const request = async (url: string, graphql: string, variables?: any) => {
       return result?.data?.data;
     })
     .catch((error: AxiosError): string => {
-      throw error.message;
+      const detailedErrorMessage = error?.response?.data?.errors
+        ?.map((error: { message: string }) => {
+          return error.message;
+        })
+        .join(". ");
+      throw detailedErrorMessage || error.message;
```

Comment:
not really a comment, more of a discussion point: we should really have uniform error handling. Sometimes we throw, sometimes we console.log, sometimes we put it on the scope.

Reply:
agreed